Esterni/PianoProgetto/informazioni.tex

```diff
@@ -0,0 +1,22 @@
+\newcommand{\Titolo}{Piano Di Progetto}
+
+\newcommand{\Gruppo}{MercurySeven }
+
+\newcommand{\VersioneDocumento}{\versNdP}
+
+\newcommand{\ACapoRedazione}{Tommaso Poppi
+Matteo Pagotto}
+
+\newcommand{\Verifica}{Verificatori}
+
+\newcommand{\Responsabile}{Responsabile}
+
+\newcommand{\Distribuzione}{Prof. Tullio Vardanega \newline Prof. Riccardo Cardin \newline Gruppo \Gruppo{}}
```

Comment:
Prof. Tullio Vardanega, Prof. Riccardo Cardin e "Mercury Seven" vanno tutti in corsivo

Reply:
risolto